src/sdk/sdk.js

```diff
@@ -0,0 +1,75 @@
+const https = require('https');
+const fs = require('fs');
+const unzipper = require('unzipper');
+const {exec} = require("child_process");
+const ERROR_COLOR  ='\x1b[36m%s\x1b[0m';
+
+module.exports = sdk = async (language, directory, apiKey) => {
```

Comment:
We ask for directory, but later (in checkProject and in extract) we use `lib/${language}-sdk`.
Is this correct?